### PR TITLE
Fixes for regTests on latest Omni Core and on Spock 2.0

### PR DESCRIPTION
**Pull request by msgilligan**

### CODE REVIEW

File: omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/sto/SendToOwnersTestPlanSpec.groovy [hunk 1]
```diff
@@ -20,6 +21,7 @@ import spock.util.mop.Use
  * Data driven tests for the "send to owners" transaction type
  */
 @Use(NumberCategory)
+@Ignore("Spock 2.0 doesn't seem to handle AssumptionViolatedException as we expect")
```
[the remark this answers is not in the document]
Comment (dexX7) [reply omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/sto/SendToOwnersTestPlanSpec.groovy:24]: Couldn't we just throw a different exception on the failing paths?

[the remark this answers is not in the document]
Comment (msgilligan) [reply omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/sto/SendToOwnersTestPlanSpec.groovy:24]: We need to find an exception that causes a test to show up as _ignored_, not _failed_. Right?

File: omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SimpleSendReorgSpec.groovy [hunk 2]
```diff
@@ -29,12 +30,13 @@ class SimpleSendReorgSpec extends BaseReorgSpec {
...
         when: "a new block is mined"
         clearMemPool()
-        generate()
+        generateBlocks(1)
 
         then: "the send transaction is no longer valid"
         !checkTransactionValidity(txid)
     }
 
+    @Ignore("Is failing with: ProcessNewBlock, block not accepted")
```
[the remark this answers is not in the document]
Comment (Bushstar) [reply omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SimpleSendReorgSpec.groovy:39]: Duplicate block error. After the chain is reset with invalidateblock, the newly minted block has the same hash the one previous one. This happens as they use the same coinbase address and when the new block is minted in the same second.

[the remark this answers is not in the document]
Comment (msgilligan) [reply omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SimpleSendReorgSpec.groovy:39]: Maybe (at least temporarily) we should wait about 1 second before minting new blocks in these tests?

[the remark this answers is not in the document]
Comment (Bushstar) [reply omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SimpleSendReorgSpec.groovy:39]: @msgilligan that would work. Could wait a second after any call to invalidate block.

**Comment by msgilligan** [comment]
This PR has been replaced by:

* PR #179
* PR #180
* PR #181
* PR #188
* PR #189

**Pull request closed without merging**